src/transformers/models/wav2vec2/processing_wav2vec2.py

```diff
@@ -72,6 +73,19 @@ def __call__(self, *args, **kwargs):
         """
         return self.current_processor(*args, **kwargs)
 
+    def _retrieve_tokens_with_time_stamps(
```

Comment:
Moving the entry point to the Processor is definitely a good idea, this makes the access to `stride` and `sampling_rate` less awkward than if it was completely inside the tokenizer. :slightly_smiling_face: